FastImageCache/FICImageCache.m

```diff
@@ -79,12 +79,16 @@ + (dispatch_queue_t)dispatchQueue {
 }
 
 - (id)init {
```

Comment:
Since you returned instancetype from your new initializer, why not use it here as well?